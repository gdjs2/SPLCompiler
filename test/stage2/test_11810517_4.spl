int main() {
	int a = 10;
	if (a < 5) {
		int b = 10;
		int b;
		if (b == 10) {
			int c = 20;
		}
	}
	return 0;
}