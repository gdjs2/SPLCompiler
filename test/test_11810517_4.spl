int main(){
    char 123abc = 0
}