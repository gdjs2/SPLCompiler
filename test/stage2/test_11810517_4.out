Error type 3 at Line 5: variable "b" is redefined.